pyproject.toml

```diff
@@ -9,7 +9,7 @@ license = {text = "MPLv2"}
 requires-python = ">=3.8"
 dependencies = [
     "attrs>=21.2.0",
-    "sentry-sdk>=1.5.0",
+    "sentry-sdk>=1.5.0,<2.0.0",
```

Comment:
Need this so tests pass. Otherwise they fail because it installs sentry-sdk>2.0.0 which changed things.